apps/charterafrica/src/payload/collections/Partners.js

```diff
@@ -0,0 +1,91 @@
+import link from "../fields/link";
+import linkGroup from "../fields/linkGroup";
+
+const partnerGroup = linkGroup();
+
+const linkField = link();
+linkField.fields = [
+  {
+    name: "logo",
+    required: true,
+    localized: true,
+    type: "upload",
+    relationTo: "media",
+    label: {
+      en: "Logo",
+      pt: "Imagem",
+      fr: "Image",
+    },
+  },
+  ...linkField.fields,
+];
+
+partnerGroup.fields = [linkField];
+
+const Partners = {
+  slug: "partners",
+  labels: {
+    singular: {
+      en: "Partner",
+      fr: "Partenaire",
+    },
+    plural: {
+      en: "Partners",
+      fr: "Partenaires",
+    },
+  },
+  admin: {
+    useAsTitle: "name",
+    defaultColumns: ["name", "updatedAt"],
+  },
+  access: {
+    read: () => true,
+  },
+  fields: [
+    {
+      type: "row",
+      fields: [
+        {
+          name: "name",
+          label: {
+            en: "Name",
+            fr: "Nom",
+            pt: "Nome",
+          },
+          type: "text",
+          localized: true,
+          required: true,
+          admin: {
+            width: "50%",
+          },
+        },
+      ],
+    },
+    {
+      type: "row",
+      fields: [
+        {
+          name: "description",
+          label: {
+            en: "Description",
+            fr: "La description",
+            pt: "Descrição",
+          },
+          type: "richText",
+          admin: {
+            elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+          },
+          localized: true,
+          required: true,
+        },
+      ],
+    },
+    {
+      type: "array",
+      name: "partners",
+      fields: [partnerGroup],
+    },
+  ],
+};
+
+export default Partners;
```

Comment:
I think we only need 👇🏽 here (also lets review and merge #303 so that we can use the custom `richText` field that handles links for us).

```suggestion
import linkGroup from "../fields/linkGroup";

const Partners = {
  slug: "partners",
  labels: {
    singular: {
      en: "Partner",
      fr: "Partenaire",
    },
    plural: {
      en: "Partners",
      fr: "Partenaires",
    },
  },
  admin: {
    useAsTitle: "title",
    defaultColumns: ["name", "updatedAt"],
  },
  access: {
    read: () => true,
  },
  fields: [
    {
      name: "title",
      label: {
        en: "Title",
        fr: "Titre",
        pt: "Título",
      },
      type: "text",
      localized: true,
      required: true,
    },
    {
      name: "partners",
      type: "array",
      min: 1,
      fields: [
        {
          name: "name",
          label: {
            en: "Name",
            fr: "Nom",
            pt: "Nome",
          },
          type: "text",
          required: true,
          localized: true,
        },
        {
          name: "logo",
          required: true,
          type: "upload",
          relationTo: "media",
          label: {
            en: "Logo",
            pt: "Imagem",
            fr: "Image",
          },
        },
        linkGroup(),
      ],
      required: true,
    },
    {
      name: "description",
      label: {
        en: "Description",
        fr: "La description",
        pt: "Descrição",
      },
      type: "richText",
      admin: {
        elements: [],
      },
      localized: true,
      required: true,
    },
  ],
};

export default Partners;
```

Reply:
Done